### ISSUE TITLE
ERR_FILE_NOT_FOUND bundle (.css & .js)

### ISSUE THREAD

**Issue by mumairofficial** [pastahito/electron-react-webpack#3]
Hi,

After following all steps to setup and upon `npm start dev` also tried `yarn start dev` getting error in console.

```
// symbol <...> to hide long path string
GET <...>/electron-react-webpack/app/build/bundle.css net::ERR_FILE_NOT_FOUND
GET <...>/electron-react-webpack/app/build/bundle.js net::ERR_FILE_NOT_FOUND
```

Can you please suggest any resolution?

**Comment by pastahito** [comment]
@mumairofficial I'm not able to reproduce such error. I followed the [install](https://github.com/pastahito/electron-react-webpack#install) and [usage](https://github.com/pastahito/electron-react-webpack#usage) steps and it worked.

In [usage](https://github.com/pastahito/electron-react-webpack#usage) section, the commands are not the ones the triggered such error but `npm run dev` and `npm start`, have you tried them? Does it work now?

**Comment by mumairofficial** [comment]
okay I got this; problem was whn I run `npm start` expecting following both commands will work with single one.

- you have build it first with `npm run dev` it generates build directory
- then run `npm start` which is `electron .`

We should configure both command in single with watch mode :-)

**Comment by pastahito** [comment]
Fixed in 083aa77c6396fca17b676546c7ebded9696d44c8 by @konsumer under PR #4. Now it builds and serve with a `npm start` only.
